nossas/bonde-bot
# Feature mass message

# Related issues
- Form to sent mass message to facebook users #5

# How to test
- Access the `/` route
- If you are not authenticated, it should redirects to the `/login` route

| desktop | mobile |
|---|---|
| <img width="1674" alt="screen shot 2017-07-11 at 13 45 31" src="https://user-images.githubusercontent.com/5435389/28079563-45d6378c-663f-11e7-8877-2a768d25b37e.png"> | <img width="397" alt="screen shot 2017-07-11 at 13 48 00" src="https://user-images.githubusercontent.com/5435389/28079693-c0425ee2-663f-11e7-8980-ecf1162ad991.png"> |

- Otherwise, it should render the mass message page by default:

| desktop | mobile |
|---|---|
| <img width="1674" alt="screen shot 2017-07-12 at 16 46 23" src="https://user-images.githubusercontent.com/5435389/28136997-007082c2-6722-11e7-9ab4-07f2c241ae9b.png"> | ![localhost-5000-mass-message-](https://user-images.githubusercontent.com/5435389/28137010-0f1b6558-6722-11e7-8fdf-ad69805e3623.png) |

- Try to submit the form with the message field empty
- It should display a validation message:
<img width="711" alt="screen shot 2017-07-12 at 16 50 57" src="https://user-images.githubusercontent.com/5435389/28137069-556a5dde-6722-11e7-9199-a71c812d05ab.png">

- Fill the message field and submit the form
- It should display a confirmation message:
<img width="567" alt="screen shot 2017-07-12 at 16 52 36" src="https://user-images.githubusercontent.com/5435389/28137124-8a4c283e-6722-11e7-85aa-2205ab3c73a1.png">

- Click the **Cancel** button to cancel the form's submit
- Or click the **OK** button to send the message
- It should send the message to all recipients that are selected:
<img width="416" alt="screen shot 2017-07-12 at 16 54 41" src="https://user-images.githubusercontent.com/5435389/28137211-d79dc7f0-6722-11e7-85d9-80a37216ac82.png">
